sensibo/unified.js

```diff
@@ -292,6 +302,43 @@ module.exports = {
 		}
 
 		return acState
-	}
+	},
 
+	sensiboFormattedClimageReactState: (device, state) => {
+    
+		smartModeState = state.smartMode
+
+		const climateReactState = {
+			enabled: smartModeState.enabled,
+			type: smartModeState.type,
+			lowTemperatureThreshold: smartModeState.lowTemperatureThreshold,
+			lowTemperatureState:
+			{
+				on: smartModeState.lowTemperatureState.on,
+				swing: smartModeState.lowTemperatureState.swing === 'SWING_ENABLED' ? 'rangeFull' : 'stopped',
```

Comment:
unified.sensiboFormattedClimageReactState() just formats "state.smartMode" to Sensibo's format, based on the values of state.smartMode, which are updated by StateManager.updateClimateReact() which is in turn called every time a relevant AC setting is changed in StateManager (e.g. ACActive, PureActive, TargetHeaterCoolerState, CoolingThresholdTemperature, HeatingThresholdTemperature, ACSwing, ACRotationSpeed, PureRotationSpeed ...)

I updated unified.sensiboFormattedClimageReactState() with the same logic that exists in unified.sensiboFormattedACState(). This is a temporary code should be refactored before release since it has a lot of duplication (both within the function and with the "sensiboFormattedACState" function).

There is a question however if/what should be the value of "state.horizontalSwing" if the device is capable of a "threeDimensionalSwing". This might necessitate changes in both ac & climate formatting functions.